features/article_have_view_page.feature

```diff
@@ -8,6 +8,8 @@ Background:
     | headline                  | content     |
     | The awesome article       | Lorem ipsum dolor sit amet, consectetur adipisicing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. |
 
+  And user is signed in
+
```

Comment:
I see you use the role Subscriber in the scenario below. (I know this is legacy). Please revisit the scenario title and adjust it or create a PT ticket for it.